docs/contributor_guide/contributing_example_notebooks.rst

```diff
@@ -17,16 +17,60 @@ A good example notebook exhibits the following attributes:
    Avoids abstraction traps.
    See `Selbst et al. (2020) <https://andrewselbst.files.wordpress.com/2019/10/selbst-et-al-fairness-and-abstraction-in-sociotechnical-systems.pdf>`_.
 4. **Substantiated**: Discusses trade-offs and compares alternatives.
-   Describes why using particular Fairlearn functionalities makes sense.
+   Describes why using particular Fairlearn functionalities makes sense if
+   Fairlearn is used.
 5. **For developers**: Speaks the language of developers and data scientists.
    Considers real practitioner needs.
-   Fits within the lifecycle of real practioner work.
+   Fits within the lifecycle of real practitioner work.
    See `Holstein et al (2019) <https://arxiv.org/pdf/1812.05239.pdf>`_,
    `Madaio et al. (2020) <http://www.jennwv.com/papers/checklists.pdf>`_.
 
 Please keep these in mind when creating, discussing, and critiquing examples.
 
-If you'd like to collaborate, please :ref:`reach out <communication>`.
+Process
+^^^^^^^
+
+All current efforts are tracked through items in the corresponding
+`Project Board <https://github.com/fairlearn/fairlearn/projects/3>`_.
+If you'd like to suggest a different kind of project, please
+`open a new issue <https://github.com/fairlearn/fairlearn/issues/new/choose>`_
+if you cannot find a similar one that is already tracked in the project board.
+If you'd like to collaborate with others from the community, please
+:ref:`reach out <communication>` to share your idea.
+
+In the issue, make sure to outline your goals, describe your audience, and
+provide a high-level overview of what you would like to show.
+Note that the examples do not need to contain code.
+In some application contexts it may be preferable not to build a
+technical system, so a textual explanation is perfectly acceptable.
+All contributions should abide by the guidelines outlined above, though.
+
+Once the issue is on GitHub, members from the community can respond with
+questions, comments, and perhaps express their interest in joining the
+project. Once the preliminary questions are sorted out, the participants of
+the project can
+`open a pull request <https://github.com/fairlearn/fairlearn/compare>`_.
+Everyone from the community is encouraged to review the pull request and
+provide feedback. The acceptance process is identical to all other
+contributions as outlined :ref:`here <development_process>`, except that
+the additional criteria at the top of this page apply additionally.
+
+Whether and how project members communicate is up to the members themselves.
+This could include any of the following:
+
+- a dedicated channel on [Gitter](https://gitter.im/fairlearn) in the
+  Fairlearn community
+- a Slack channel or Teams team
+- email
+- recurring or ad-hoc meetings
```

Comment:
I think we should encourage asynchronous communication on github, with the added possibility of whatever public chat channel we have. Otherwise people from certain timezones and people who are not involved in the discussion are excluded easily.

Reply:
Indeed. I think the "Discussions" might actually lend itself to this (more so that issues since they are linear). Let me update that. For this particular point we really want the individual groups to make those decisions rather than us forcing anything onto people. If there are a few people who want to have this particular discussion over email they can certainly do that.

Reply:
Updated this to encourage people to use the more inclusive async communication. Thanks for suggesting!

Reply:
I would still argue that this should be strongly discourages since it excludes everybody who's not on that email thread.